jmeter-test/4node/deploy-erc1155-expand-contract-private.jmx

```diff
@@ -32,7 +32,7 @@
ffffffffffffffff821117156120fd576120fc6121dc565b5b80604052505050565b600061211182612057565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff8214156121445761214361214f565b5b600182019050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b600060033d111561222a5760046000803e612227600051612257565b90505b90565b600080fd5b600080fd5b600080fd5b600080fd5b600080fd5b6000601f19601f8301169050919050565b60008160e01c9050919050565b7f455243313135353a207472616e7366657220746f206e6f6e204552433131353560008201527f526563656976657220696d706c656d656e746572000000000000000000000000602082015250565b7f455243313135353a204552433131353552656365697665722072656a6563746560008201527f6420746f6b656e73000000000000000000000000000000000000000000000000602082015250565b7f455243313135353a2062616c616e636520717565727920666f7220746865207a60008201527f65726f2061646472657373000000000000000000000000000000000000000000602082015250565b7f455243313135353a2063616c6c6572206973206e6f74206f776e6572206e6f7260008201527f20617070726f7665640000000000000000000000000000000000000000000000602082015250565b7f455243313135353a207472616e7366657220746f20746865207a65726f20616460008201527f6472657373000000000000000000000000000000000000000000000000000000602082015250565b7f455243313135353a207472616e736665722063616c6c6572206973206e6f742060008201527f6f776e6572206e6f7220617070726f7665640000000000000000000000000000602082015250565b7f455243313135353a20696e73756666696369656e742062616c616e636520666f60008201527f72207472616e7366657200000000000000000000000000000000000000000000602082015250565b7f455243313135353a2073657474696e6720617070726f76616c2073746174757360008201527f20666f722073656c660000000000000000000000000000000000000000000000602082015250565b7f455243313135353a206163636f756e747320616e6420696473206c656e67746860008201527f206d69736d617463680000000000000000000000000000000000000000000000602082015250565b7f455243313135353a2069647320616e6420616d6f756e7473206c656e6774682060008201527f6d69736d61746368000000000000000000000000000000000000000000000000602082015250565b600060443d101561258a5761260d565b612592611e93565b60043d036004823e80513d602482011167ffffffffffffffff821117156125ba57505061260d565b808201805167ffffffffffffffff8111156125d8575050505061260d565b80602083010160043d0385018111156125f557505050505061260d565b612604826020018501866120d5565b82955050505050505b90565b61261981611fed565b811461262457600080fd5b50565b61263081611fff565b811461263b57600080fd5b50565b6126478161200b565b811461265257600080fd5b50565b61265e81612057565b811461266957600080fd5b5056fea264697066735822122014de258cf14a9394bf2abe28957fef7481c356a77ff26e1868092e0c6851da6264736f6c63430008060033&quot; ${privateForField} }], &quot;id&quot;: 1 }</stringProp>
```

Comment:
what was the issue with the contract creation?

Reply:
id was at wrong place